### PR TITLE
⌗114903 Black Friday 2018 Admin Notice

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**⌗114903**](http://central.tri.be/issues/114903)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Marketing.php:5](https://github.com/moderntribe/tribe-common/blob/613e2cbe20a0ad7723fa67d031dfa8d56cf75e23/src/Tribe/Admin/Notice/Marketing.php#L5)
:warning: | Comment refers to a TODO task | [Marketing.php:6](https://github.com/moderntribe/tribe-common/blob/613e2cbe20a0ad7723fa67d031dfa8d56cf75e23/src/Tribe/Admin/Notice/Marketing.php#L6)
:warning: | Comment refers to a TODO task | [Marketing.php:7](https://github.com/moderntribe/tribe-common/blob/613e2cbe20a0ad7723fa67d031dfa8d56cf75e23/src/Tribe/Admin/Notice/Marketing.php#L7)
:warning: | Comment refers to a TODO task | [Marketing.php:8](https://github.com/moderntribe/tribe-common/blob/613e2cbe20a0ad7723fa67d031dfa8d56cf75e23/src/Tribe/Admin/Notice/Marketing.php#L8)
:warning: | Comment refers to a TODO task | [Marketing.php:9](https://github.com/moderntribe/tribe-common/blob/613e2cbe20a0ad7723fa67d031dfa8d56cf75e23/src/Tribe/Admin/Notice/Marketing.php#L9)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Marketing.php:5](https://github.com/moderntribe/tribe-common/blob/c36a62b71f6b8aeee2083721d019aa5a35dcd4dc/src/Tribe/Admin/Notice/Marketing.php#L5)
:warning: | Comment refers to a TODO task | [Marketing.php:6](https://github.com/moderntribe/tribe-common/blob/c36a62b71f6b8aeee2083721d019aa5a35dcd4dc/src/Tribe/Admin/Notice/Marketing.php#L6)
:warning: | Comment refers to a TODO task | [Marketing.php:7](https://github.com/moderntribe/tribe-common/blob/c36a62b71f6b8aeee2083721d019aa5a35dcd4dc/src/Tribe/Admin/Notice/Marketing.php#L7)
:warning: | Comment refers to a TODO task | [Marketing.php:8](https://github.com/moderntribe/tribe-common/blob/c36a62b71f6b8aeee2083721d019aa5a35dcd4dc/src/Tribe/Admin/Notice/Marketing.php#L8)
:warning: | Comment refers to a TODO task | [Marketing.php:9](https://github.com/moderntribe/tribe-common/blob/c36a62b71f6b8aeee2083721d019aa5a35dcd4dc/src/Tribe/Admin/Notice/Marketing.php#L9)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Marketing.php:5](https://github.com/moderntribe/tribe-common/blob/28d798aae24725a8637e43b145a19b2a6edbc0a9/src/Tribe/Admin/Notice/Marketing.php#L5)
:warning: | Comment refers to a TODO task | [Marketing.php:6](https://github.com/moderntribe/tribe-common/blob/28d798aae24725a8637e43b145a19b2a6edbc0a9/src/Tribe/Admin/Notice/Marketing.php#L6)
:warning: | Comment refers to a TODO task | [Marketing.php:7](https://github.com/moderntribe/tribe-common/blob/28d798aae24725a8637e43b145a19b2a6edbc0a9/src/Tribe/Admin/Notice/Marketing.php#L7)
:warning: | Comment refers to a TODO task | [Marketing.php:8](https://github.com/moderntribe/tribe-common/blob/28d798aae24725a8637e43b145a19b2a6edbc0a9/src/Tribe/Admin/Notice/Marketing.php#L8)
:warning: | Comment refers to a TODO task | [Marketing.php:9](https://github.com/moderntribe/tribe-common/blob/28d798aae24725a8637e43b145a19b2a6edbc0a9/src/Tribe/Admin/Notice/Marketing.php#L9)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [tribe-bf-2018-tec.php:18](https://github.com/moderntribe/tribe-common/blob/28d798aae24725a8637e43b145a19b2a6edbc0a9/src/admin-views/notices/tribe-bf-2018-tec.php#L18)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Marketing.php:5](https://github.com/moderntribe/tribe-common/blob/caa1aee13dd28585889fc6e3f175efbef9411171/src/Tribe/Admin/Notice/Marketing.php#L5)
:warning: | Comment refers to a TODO task | [Marketing.php:6](https://github.com/moderntribe/tribe-common/blob/caa1aee13dd28585889fc6e3f175efbef9411171/src/Tribe/Admin/Notice/Marketing.php#L6)
:warning: | Comment refers to a TODO task | [Marketing.php:7](https://github.com/moderntribe/tribe-common/blob/caa1aee13dd28585889fc6e3f175efbef9411171/src/Tribe/Admin/Notice/Marketing.php#L7)
:warning: | Comment refers to a TODO task | [Marketing.php:8](https://github.com/moderntribe/tribe-common/blob/caa1aee13dd28585889fc6e3f175efbef9411171/src/Tribe/Admin/Notice/Marketing.php#L8)
:warning: | Comment refers to a TODO task | [Marketing.php:9](https://github.com/moderntribe/tribe-common/blob/caa1aee13dd28585889fc6e3f175efbef9411171/src/Tribe/Admin/Notice/Marketing.php#L9)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [tribe-bf-2018-tec.php:18](https://github.com/moderntribe/tribe-common/blob/caa1aee13dd28585889fc6e3f175efbef9411171/src/admin-views/notices/tribe-bf-2018-tec.php#L18)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [tribe-bf-2018-tec.php:18](https://github.com/moderntribe/tribe-common/blob/220ef1930a726440737ebbd2faeaae37faab5166/src/admin-views/notices/tribe-bf-2018-tec.php#L18)

> via jenkins/codesniffer-phpcs

**Comment by ggwicz** [comment]
There will likely be some more design tweaks incoming, but we don't need to wait for those to get this main code merged. This is ready for code review.

File: src/Tribe/Admin/Notices.php [hunk 1]
```diff
@@ -318,6 +318,7 @@ public function is_rendered_html( $slug, $html ) {
 	 * @return boolean
 	 */
 	public function has_user_dimissed( $slug, $user_id = null ) {
+		write_log( $slug, 'slug' );
```
Comment (Camwyn) [inline src/Tribe/Admin/Notices.php]: Are we intending to write to customer logs?

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for **, but this was pull requested against *release/M18.16*.

**Comment by ggwicz** [comment]
Addressed Stephen's question; removing `question` tag as a result, and tagging this `merge` since he approved it for merging.

**Comment by ggwicz** [comment]
Submitted some final style tweaks based on feedback from Jen and Kyle. This is ready for a final review :-)

**Pull request merged**